components/network/isc-dhcp/Makefile

```diff
@@ -20,16 +20,16 @@
 #
 # Copyright (c) 2011, 2019, Oracle and/or its affiliates. All rights reserved.
 # Copyright (c) 2020, Michal Nowak
+# Copyright (c) 2022, Nona Hansel
 #
 
 BUILD_BITS=		64
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		isc-dhcp
-COMPONENT_VERSION=	4.4.2
-COMPONENT_REVISION=	2
-IPS_COMPONENT_VERSION=	4.4.2.0.0
+COMPONENT_VERSION=	4.4.3
+IPS_COMPONENT_VERSION=	4.4.3.0.0
```

Comment:
I am not sure whether the use of IPS_COMPONENT_VERSION makes sense here. It is most probably a relic from times where COMPONENT_VERSION had some illegal characters. You should remove it.

Reply:
Removed, thanks.